docker/Dockerfile

```diff
@@ -146,6 +146,7 @@ COPY "./docker" "./docker"
 COPY "./docs" "./docs"
 COPY "./examples" "./examples"
 COPY "./models" "./models"
+COPY "./morpheus/data" "./morpheus/data"
```

Comment:
This directory gets installed now with the `morpheus` python package. If you run:

```bash
$ python -c "import morpheus;print(morpheus.DATA_DIR)"
$HOME/Repos/morpheus/morpheus-dev/morpheus/data
```
Is this path used by scripts anywhere?

Reply:
Let me double check things once we have a working release build. I'll validate that our external instructions are correct.

Reply:
Release build should be working as of #164